fluentd/configs.d/openshift/es-copy-config.conf

```diff
@@ -28,4 +28,5 @@
       # queue limit is hit - 'block' will halt further reads and keep retrying to flush the
       # buffer to the remote - default is 'exception' because in_tail handles that case
       buffer_queue_full_action "#{ENV['BUFFER_QUEUE_FULL_ACTION'] || 'exception'}"
+      ssl_verify "#{ENV['ES_COPY_SSL_VERIFY'] || 'true'}"
```

Comment:
@nhosoi @portante ok, how about this plan:

- for 3.7, submit a PR that removes the ES COPY feature, then submit a PR that only does the "one output for one ES" feature
- continue with this PR for 3.6 only
- we'll need to figure out another way to test without exposing `ssl_verify`